src/tempods/pages/__init__.py

```diff
@@ -1,21 +1,178 @@
+        tempo_data = RemoteGeoData_ArcGISImageServer(asdc_url, name="TEMPO")
+        app.add_data(tempo_data)
+
+        # Our remote dataset does not have real components representing latitude and longitude. We link to the only components
+        # it does have so that we can display this on the same viewer without trigger and IncompatibleAttribute error
+        app.add_link(app.data_collection["Power_Plants"], 'Longitude',  app.data_collection["TEMPO"], 'Pixel Axis 0')
+        app.add_link(app.data_collection["Power_Plants"], 'Latitude', app.data_collection["TEMPO"], 'TEMPO_NO2_L3_V03_HOURLY_TROPOSPHERIC_VERTICAL_COLUMN_BETA')
+
+        big = (powerplant_data['Install_MW'] > 100)
+        med = (powerplant_data['Install_MW'] > 10) & (powerplant_data['Install_MW'] <= 100)
+        small = (powerplant_data['Install_MW'] <= 10)
+        powerplant_data.add_component(big*9 + med*4 + small*1, label='Size_binned')
+
+        map = cast(IPyLeafletMapViewer, app.new_data_viewer(IPyLeafletMapViewer, data=tempo_data, state=map_state, show=False))
+        map.figure_widget.layout = {"width": "100%", "height": "500px"}
+        map.map.panes = {"labels": {"zIndex": 650}}
+        map.map.add(TileLayer(url=stadia_labels_url, pane='labels'))
+        map.map.add(geo_json)
+
+        timeseries = cast(TimeSeriesViewer, app.new_data_viewer(TimeSeriesViewer, data=tempo_data, show=False))
+        timeseries.figure.axes[1].label_offset = "-50"
+        timeseries.figure.axes[1].tick_format = ".0f"
+        timeseries.figure.axes[1].label = "Amount of NO2 (10^14 molecules/cm^2)"
+
+        timeseries.figure.axes[0].label_offset = "40"
+        timeseries.figure.axes[0].label = "Time (UTC)"
+
+        timeseries.figure.axes[0].label_color = "white"
+        timeseries.figure.axes[1].label_color = "white"
+
+        timeseries.figure.axes[0].tick_style = {"stroke": "white"}
+        timeseries.figure.axes[1].tick_style = {"stroke": "white"}
+
+        return app, map, timeseries
+
+    gjapp, map_viewer, timeseries_viewer = solara.use_memo(_glue_setup, dependencies=[])
+    powerplant_data = gjapp.data_collection["Power_Plants"]
+    tempo_data = gjapp.data_collection["TEMPO"]
+
+    def convert_from_milliseconds(milliseconds_since_epoch):
+        """Converts milliseconds since epoch to a date-time string in 'YYYY-MM-DDTHH:MM:SSZ' format."""
+        dt = datetime.fromtimestamp((milliseconds_since_epoch)/ 1000, tz=timezone(offset=timedelta(hours=0), name="UTC"))
+        date_time_str = dt.strftime('%H:%M')
+        return date_time_str
+
+    time_index, set_time_index = solara.use_state(0)
+    current_date, set_current_date = solara.use_state("2024-10-15")
+    time_values, set_time_values = solara.use_state(tempo_data.get_time_steps(current_date))
+
+    def update_image(index):
+        print(f"update_image: {index}")
+        set_time_index(index)
+        timestep = time_values[index]
```

Comment:
This needs to be:

`timestep = time_values[index].item()`

to cast the solara-returned `np.int64` down to an `int` (otherwise ipyleaflet casts the `np.int64` to a string, which breaks things)